### ISSUE TITLE
Problem: there are multiple places to set timezone and no docs to describe these places

### ISSUE THREAD

**Issue by kellyannewithane** [archivematica/Issues#291]
**Please describe the problem you'd like to be solved.**
Currently there are multiple places where timezone can be set - the dashboard, the stage service, the database, and elasticsearch (and possibly others?).  Additionally, the docs don't list where these places are or describe how to change timezones in installed environments.

**Describe the solution you'd like to see implemented.**
Ideally, it would be great to set the time zone for an AM environment configurable in 1 place.

At the very least we could provide some instructions for current functionality so that it's clear to others where and how time zones can/should be changed.

**Describe alternatives you've considered.**

**Additional context**

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by ThomasEdvardsen** [comment]
Is this about timezone in logfiles or in GUI? Would like to have the same timezone across all logfiles :)

**Comment by ross-spencer** [comment]
Hi @ThomasEdvardsen, @kellyannewithane is away this week but I believe it's across the board. This task is on my TODO list but is a little bit lower priority right now with release and client work. I need to refine the story here a bit more too. Are you seeing different results across different servers?

**Comment by ThomasEdvardsen** [comment]
@ross-spencer we have different timezones in the log files on the same server. Especially the Storage Server seems to log UTC -07:00, and we are using UTC + 01:00 here in Norway.

**Comment by ross-spencer** [comment]
Thanks @ThomasEdvardsen that's useful information. Will start digging into this some more.

**Comment by ThomasEdvardsen** [comment]
I have managed to get dashboard, MCP Server, MCP Client and Storage Service to log with the same timezone now. It's not possible with configuration parameters, so had to change source code.

Dashboard: [src/settings/base.py](https://github.com/artefactual/archivematica/blob/stable/1.8.x/src/dashboard/src/settings/base.py#L187)
MCP Server: [lib/settings/common.py](https://github.com/artefactual/archivematica/blob/stable/1.8.x/src/MCPServer/lib/settings/common.py#L143)
MCP Client: [lib/settings/common.py](https://github.com/artefactual/archivematica/blob/stable/1.8.x/src/MCPClient/lib/settings/common.py#L152)
Storage Service: [settings/base.py](https://github.com/artefactual/archivematica-storage-service/blob/stable/0.14.x/storage_service/storage_service/settings/base.py#L65)

Those should be possible to set i config, so the settings are the same after upgrading the software.

**Comment by scollazo** [comment]
As per @ThomasEdvardsen note, I updated the TIME_ZONE hardcoded values so now they can be configured using the system environment variables.

**Comment by sevein** [comment]
The scope of this issue I think goes beyond the TIME_ZONE setting, i.e. we may have to make recommendations in other areas of the system, like the system timezone settings or defaults in services such MySQL or Elasticsearch. But Santi has submitted PRs (https://github.com/artefactual/archivematica/pull/1583 and https://github.com/artefactual/archivematica-storage-service/pull/525) so TIME_ZONE (the app-specific setting provided by Django) is configurable which is a good start.

**Comment by sallain** [comment]
@sevein @scollazo sounds like filing a more specific issue that we can pin the PR to would make sense?

**Comment by sevein** [comment]
Good idea @sallain, I've created https://github.com/archivematica/Issues/issues/1143 and updated the relationships.

**Comment by replaceafill** [comment]
This issue is being closed as part of a bulk closure process. Please click [here](https://groups.google.com/u/1/g/archivematica/c/Sq7uw1zloTA) for further information.
